packages/core/src/index.ts

```diff
@@ -496,8 +506,9 @@ class Hopr extends EventEmitter {
   public async stop(): Promise<void> {
     this.status = 'DESTROYED'
     clearTimeout(this.checkTimeout)
-    await Promise.all([this.heartbeat.stop(), (await this.startedPaymentChannels()).stop()])
-
+    verbose('Stopping heartbeat & indexer')
+    await Promise.all([this.heartbeat.stop(), await this.chain.stop()])
```

Comment:
`await` under `Promise.all` ?

Reply:
Are there any issues with this? FWIW this is a refactor from the prev. line (`#499`)

Reply:
`await` resolve a Promise already, why would you put it under `Promise.all` ? Either remove `await` (makes more sense) or move it to a separate line (less sense). Otherwise it just confuses the reader

Reply:
Solved in b1716acf0727ea320e306e500886495badf802ef